docs/CHANGELOG.md

```diff
@@ -23,10 +24,27 @@
 * [Added] Add `--meta` argument to `push` ([#1793](https://github.com/quiltdata/quilt/issues/1793))
 * [Fixed] Fix crash in `list-packages` ([#1852](https://github.com/quiltdata/quilt/issues/1852))
 
+## Catalog, Lambdas
+* [Added] Ability to preview larger Jupyter notebooks; warning when cells are
+elided ([#1823](https://github.com/quiltdata/quilt/issues/1823),
```

Comment:
Unfortunately GitBook renders each line in a list item as a separate paragraph like here:
https://docs.quiltdata.com/#quilt-in-action

Not sure if there is a way to fix it and if we should fix it. Instead we might want to try to switch to [readthedocs.io](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx).

Reply:
This is silly, and a bit of Google Fu finds no workarounds, but I think we should live with it for now.